src/main/resources/META-INF/plugin.xml

```diff
@@ -80,24 +80,19 @@
   ]]>
   </change-notes>
 
-  <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for description -->
-  <idea-version since-build="173.0"/>
-
   <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html
        on how to target different products -->
   <depends>com.intellij.modules.lang</depends>
+  <depends>com.intellij.modules.platform</depends>
```

Comment:
Why is it required ?

Reply:
This is not required. Will remove it.

Reply:
I will remove it, added it during the debugging session.